### PR TITLE
bumping versions rc9

### PR DESCRIPTION
**Pull request by erickzanardo**
# Description

Bumping versions to rc9

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors and are passing (See [Contributor Guide]).
- [x] I updated/added relevant documentation (doc comments with `///`) and updated/added examples in `doc/examples`.
- [x] I have formatted my code with `./scripts/format.sh` and the Flame analyzer (`./scripts/analyze.sh`) does not report any problems.
- [x] I read and followed the [Flutter Style Guide].
- [ ] I have added a description of the change under `[next]` in `CHANGELOG.md`.
- [ ] I removed the `Draft` status, by clicking on the `Ready for review` button in this PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flame users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in `CHANGELOG.md`).
- [x] No, this is *not* a breaking change.

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database]. Indicate, which of these issues are resolved or fixed by this PR.*

[issue database]: https://github.com/flame-engine/flame/issues
[Contributor Guide]: https://github.com/flame-engine/flame/blob/main/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo

### CODE REVIEW

File: packages/flame/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # CHANGELOG
 
-## [next]
+## [1.0.0-rc9]
```
Comment (luanpotter) [inline packages/flame/CHANGELOG.md:3]: I normally keep the next empty to avoid further conflicts. (not worth changing imo just a thought for the future)

**Review by luanpotter — APPROVED** [review]
LGTM.

we don't have that doc/*.md file with the version anymore right?

also don't forget to tag!

File: packages/flame/CHANGELOG.md (hunk 1 above)
Comment (erickzanardo) [reply packages/flame/CHANGELOG.md:3]: I will add a commit after I generate the tag, so the next don't show on pub

Comment (luanpotter) [reply packages/flame/CHANGELOG.md:3]: ahh didn't know it would show up there. I won't do that anymore then haha

**Pull request merged**